src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
+      <div class="star start">
+      </div>
```

Comment:
Fix it everywhere, please
<img width="871" alt="image" src="https://user-images.githubusercontent.com/99403198/211151597-54da420e-c54c-4bd7-9d1c-52b6025d4a62.png">
<img width="871" alt="image" src="https://user-images.githubusercontent.com/99403198/211151609-ca8836bc-aaa8-410b-aef6-37f54d4884be.png">
```suggestion
    <section class="stars stars--0">
      <div class="stars__star">
      </div>
```